index.js

```diff
@@ -4,6 +4,7 @@ const STATUSBAR_DEFAULT_HEIGHT = 20;
 const STATUSBAR_X_HEIGHT = 44;
 const STATUSBAR_IP12_HEIGHT = 47;
 const STATUSBAR_IP12MAX_HEIGHT = 47;
+const STATUSBAR_IP14PRO_HEIGHT = 49;
```

Comment:
1. only 14pro？how about 14 pro max or 14 plus
2. is 49 correct？

Reply:
IP14PRO HEIGHT = 59, I updated it after I forked it, https://www.npmjs.com/package/react-native-safearea-height